src/marin/execution/executor.py

```diff
@@ -361,6 +376,23 @@ class ExecutorInfo:
     description: str | None
     steps: list[ExecutorStepInfo]
 
+    @cached_property
+    def cached_dict(self) -> dict[str, Any]:
```

Comment:
maybe `_cached_dict` so it's clear it's not public api?

Reply:
I ended up with a simpler approach: I just convert the executor_info once and then reuse the items from it.